avm/res/net-app/net-app-account/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -22,6 +23,11 @@ param serviceShort string = 'nanaawaf'
 @description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
 param namePrefix string = '#_namePrefix_#'
 
+@description('Optional. The tags to apply to all resources.')
+param tags object = {
+  service: 'netapp'
+}
```

Comment:
The tags should be part of the test itself, not in a variable or parameter. For context: This is not wrong - but - our utilities extract & format the `testDeployment` block of each test folder into the readme. If a value is not in that block, it's also not going to be in the readme. Instead it would just show `tags: tags`.

Reply:
Curiously enough, it failed with the error you're mentioning in the run right after: https://github.com/Azure/bicep-registry-modules/actions/runs/9618668724, AND the run before: https://github.com/Azure/bicep-registry-modules/actions/runs/9586083119/job/26433550470 . And there was no code change in the middle. So that doens't make much sense

Reply:
Thanks. Fixed.